packages/transaction-controller/src/TransactionController.ts

```diff
@@ -1399,10 +1401,12 @@ export class TransactionController extends BaseControllerV1<
     });
   }
 
+  // NOTE(JL): Where is this called?
   startIncomingTransactionProcessing() {
     this.incomingTransactionHelper.start();
   }
 
+  // NOTE(JL): Where is this called?
   stopIncomingTransactionProcessing() {
     this.incomingTransactionHelper.stop();
```

Comment:
These appear to be duplicates of 
```
 startIncomingTransactionPolling() {
    this.incomingTransactionHelper.start();
  }

  stopIncomingTransactionPolling() {
    this.incomingTransactionHelper.stop();
  }
  ```
  on lines 834-840

Reply:
These can probably be deleted?

Reply:
https://github.com/search?q=org%3AMetaMask%20stopIncomingTransactionProcessing&type=code

looks like this method isn't referenced by any repo in the entire metamask org

Reply:
done https://github.com/MetaMask/core/pull/3762/commits/b60310773bf5e1f390de25ed78fd4e8ade27ed23